e2e/cypress/integration/add_user_to_channel_from_profile_popover/profile_popover_spec.js

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @profile_popover
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Profile popover', () => {
+    let testTeam;
+    let testUser;
+    let otherUser;
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user}) => {
+            testTeam = team;
+            testUser = user;
+            cy.apiCreateUser().then(({user: secondUser}) => {
+                otherUser = secondUser;
+                cy.apiAddUserToTeam(testTeam.id, secondUser.id);
+            });
+        });
+    });
+
+    it('MM-T2 Add user — Error if already in channel', () => {
+        // [#] Do the system console setup
+        cy.apiAdminLogin();
+        cy.visit('/admin_console/user_management/permissions/system_scheme');
+        cy.findByTestId('all_users-public_channel-checkbox').scrollIntoView().should('be.visible').click();
+
+        // Expand the section
+        cy.get('#all_users-public_channel > .fa').should('be.visible').click();
+
+        // [*] Verify that all the sub-checkboxes are enabled
+        cy.findByTestId('all_users-public_channel-create_public_channel-checkbox').should('have.class', 'checked');
+        cy.findByTestId('all_users-public_channel-manage_public_channel_properties-checkbox').should('have.class', 'checked');
+        cy.findByTestId('all_users-public_channel-manage_public_channel_members_and_read_groups-checkbox').should('have.class', 'checked');
+        cy.findByTestId('all_users-public_channel-delete_public_channel-checkbox').should('have.class', 'checked');
+
+        cy.findByTestId('all_users-private_channel-checkbox').scrollIntoView().should('be.visible').click();
```

Comment:
Could you check please check again? On click, "Manage Private Channels" turns into uncheck for me, and so, the next step failed.

Reply:
Works fine for me. By default, we have the intermediate level with 3 permissions on and one off. And clicking it once, turns everything on.

Reply:
Are you on latest master? Not sure if https://github.com/mattermost/mattermost-server/pull/15370 changed such default.

Reply:
Just checked out latest master, and it works.

Reply:
Then, I'm fine as is. Will just monitor once merged. Thanks for checking.